plugins/woocommerce/tests/performance/config.js

```diff
@@ -9,6 +9,7 @@ export const admin_acc_login = __ENV.A_ACC_LOGIN || false;
 export const customer_username =
 	__ENV.C_USER || 'customer@woocommercecoree2etestsuite.com';
 export const customer_password = __ENV.C_PW || 'password';
+export const customer_user_id = __ENV.C_UID || '2';
```

Comment:
Thanks for the suggestion! The config has grown bigger as we have added more requests so I would definitely be open to ways of making it better. Perhaps this could be done in a separate PR?

Reply:
Sure thing!